core/server/master/src/main/java/alluxio/underfs/UfsStatusCache.java

```diff
@@ -134,6 +134,7 @@ public Collection<UfsStatus> addChildren(AlluxioURI path, Collection<UfsStatus>
     // Update global counters for all InodeSyncStream
     int sizeChange = children.size() - (previousStatuses != null ? previousStatuses.size() : 0);
     DefaultFileSystemMaster.Metrics.UFS_STATUS_CACHE_CHILDREN_SIZE_TOTAL.inc(sizeChange);
+    DefaultFileSystemMaster.Metrics.UFS_STATUS_CACHE_SIZE_TOTAL.inc(sizeChange);
```

Comment:
This change is not needed because the increment happens in `addStatusUnchecked()`